src/components/layout/FlexCol/FlexCol.tsx

```diff
@@ -42,15 +42,19 @@ const genAutoWidth = breakpoint => css`
   }
 `;
 
-const genRelativeWidth = (breakpoint, colWidth, cols) => css`
+const genRelativeWidth = (breakpoint: TGridBreakpoints, colWidth: TColWidth, cols: IFlexColProps['cols']) => css`
   @media (min-width: ${grid.breakpoints[breakpoint]}px) {
     display: block;
+    ${typeof colWidth === 'number' && typeof cols === 'number'
+      ? `
     flex: 0 0 ${(colWidth / cols) * 100}%;
     max-width: ${(colWidth / cols) * 100}%;
+    `
+      : null}
   }
 `;
 
-const getWidth = (breakpoint, colWidth, cols) => {
+const getWidth = (breakpoint: TGridBreakpoints, colWidth: TColWidth, cols: IFlexColProps['cols']) => {
   if (colWidth === 'hidden') {
```

Comment:
I know it's not on this scope but I think this would be better like this:

```ts
const getWidth = (breakpoint: TGridBreakpoints, colWidth: TColWidth, cols: IFlexColProps['cols']) => {
  switch (colWidth) {
    case 'hidden':
      return genHidden(breakpoint);
    case 'auto':
      return genAutoWidth(breakpoint);
    case 'fill':
      return genFillWidth(breakpoint);
    default:
      return genRelativeWidth(breakpoint, colWidth, cols);
  }
};

```

Reply:
Yup don't mind to try to refactor it 👍🏻